### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by noviluni**
Fixed some typos in the docs. 

If you prefer not to fix those in the `docs/news.rst` and/or those in the files under `sep/` for "historical reasons" I'm agree with that, just tell me or commit.

### CODE REVIEW

**Comment by elacuesta** [comment]
Closing and reopening to trigger a new build. The docs build is failing but I have seen the same error earlier, it seems like the server that holds `https://twistedmatrix.com/documents/current/objects.inv` is having some  intermittent issues.

File: docs/news.rst [hunk 1]
```diff
@@ -2512,15 +2512,15 @@ Scrapy 0.18.1 (released 2013-08-27)
 - test PotentiaDataLoss errors on unbound responses (:commit:`b15470d`)
 - Treat responses without content-length or Transfer-Encoding as good responses (:commit:`c4bf324`)
 - do no include ResponseFailed if http11 handler is not enabled (:commit:`6cbe684`)
-- New HTTP client wraps connection losts in ResponseFailed exception. fix #373 (:commit:`1a20bba`)
+- New HTTP client wraps connection lost in ResponseFailed exception. fix #373 (:commit:`1a20bba`)
 - limit travis-ci build matrix (:commit:`3b01bb8`)
 - Merge pull request #375 from peterarenot/patch-1 (:commit:`fa766d7`)
 - Fixed so it refers to the correct folder (:commit:`3283809`)
-- added quantal & raring to support ubuntu releases (:commit:`1411923`)
+- added Quantal & raring to support Ubuntu releases (:commit:`1411923`)
```
Comment (wRAR) [inline docs/news.rst]: Not sure if Ubuntu release names should be capitalized but these two are inconsistent.

Comment (noviluni) [reply docs/news.rst]: Hi @wRAR 
Totally agree, iI fixed it in a new commit (and capitalized more "scrapy" words). About Ubuntu release names capitalization it seems to be correct, as I found it in a lot of places. Example: http://old-releases.ubuntu.com/releases/

**Comment by kmike** [comment]
Thanks @noviluni!

**Pull request merged**